torchvision/models/quantization/googlenet.py

```diff
@@ -18,7 +20,13 @@
 }
 
 
-def googlenet(pretrained=False, progress=True, quantize=False, **kwargs):
+def googlenet(
+    pretrained: bool = False,
+    progress: bool = True,
+    quantize: bool = False,
+    **kwargs: Any
```

Comment:
would you mind adding a comma, at the end of the last arg line?

Reply:
Done! can you crosscheck once